src/lib.rs

```diff
@@ -121,17 +131,12 @@ fn normalise_dates(start: &str, end: &Option<&str>) -> Result<Vec<(NaiveDateTime
     // if the end is not set - use now
     let end_date = match end {
         None => now,
-        Some(end_date) => {
-            let end_date = parse_date(end_date)?;
-            // check that the date is not in the future - otherwise set it to now
-            if now.and_utc().timestamp() < end_date.and_utc().timestamp() {
```

Comment:
@jnioche Do you remember why this was using `.and_utc().timestamp()` when comparing dates?

I've removed it and it seems to work as before and tests seem to be fine...although with datetimes who knows, I wouldn't be surprised if it was there to avoid some weird DST bug :D (I'm not familiar with `chrono` so I may miss something very basic here).

It is easy to put back, but I usually prefer less unless there is a good reason for it (Less is More).

Reply:
Probably what chatGPT had suggested or copy and paste from stackoverflow :-)

Reply:
In that case I'll keep the simpler version until we, mere humans, are proven wrong and there is a reason to reintroduce ;)